openc3-cosmos-script-runner-api/app/models/running_script.rb

```diff
@@ -343,18 +302,42 @@ def self.spawn(scope, name, suite_runner = nil, disconnect = false, environment
     model = OpenC3::OfflineAccessModel.get_model(name: username, scope: scope) if username and username != ''
 
     # Load the global environment variables
+    status_environment = {}
     values = OpenC3::EnvironmentModel.all(scope: scope).values
     values.each do |env|
       process.environment[env['key']] = env['value']
+      status_environment[env['key']] = env['value']
     end
     # Load the script specific ENV vars set by the GUI
     # These can override the previously defined global env vars
     if environment
       environment.each do |env|
         process.environment[env['key']] = env['value']
+        status_environment[env['key']] = env['value']
       end
     end
 
+    script_status = OpenC3::ScriptStatusModel.new(
+      name: running_script_id.to_s, # Unique id for this script
+      state: 'spawning', # State will be spawning until the script is running (*)
+      shard: 0, # Future enhancement of script runner shards
+      filename: name, # Initial filename never changes
+      current_filename: name, # Current filename updates while we are running (*)
+      line_no: 0, # 0 means not running yet (*)
+      username: username, # username of the person who started the script
+      user_full_name: user_full_name, # full name of the person who started the script
+      start_time: start_time, # Time the script started ISO format
+      end_time: nil, # Time the script ended ISO format
+      disconnect: false, # Disconnect is set to true if the script is running in a disconnected mode
+      environment: status_environment.as_json(:allow_nan => true).to_json(:allow_nan => true), # nil or Hash of key/value pairs for environment variables
+      suite_runner: suite_runner ? suite_runner.as_json(:allow_nan => true).to_json(:allow_nan => true) : nil,
+      errors: nil, # array of errors that occurred during the script run (*)
+      pid: nil, # pid of the script process - set by the script itself when it starts
+      updated_at: nil, # Set by create/update - ISO format
+      scope: scope # Scope of the script
+    )
+    script_status.create(isoformat: true)
```

Comment:
Love the docs here!
What is the point of the `(*)` after the comments?

Reply:
The * was just notes to myself on which items to make sure get updated while running.  Will remove.